### PR TITLE
Kevin/new mode

### PR DESCRIPTION
**Pull request by RocketKevin**
-Added Easy Mode
-Added Hard Mode

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -106,6 +107,54 @@
   background: lightgray;
 }
 
+#dialog {
```
Comment (brotatotes) [inline src/App.css:110]: did you add this CSS in both PRs?

File: src/App.js [hunk 2]
```diff
@@ -112,11 +113,9 @@ function verifySolution() {
 
   for (var index = 0; index < 9; index++) {
     let verifyColBox = verifyCol(index, colSet, rowSet);
-    let verifyRowBox = verifyRow(index, rowSet);
-    console.log("index: " + index + ", verifyRowBox: " + verifyRowBox);
-    
+    let verifyRowBox = verifyRow(index, rowSet);
```
Comment (brotatotes) [inline src/App.js:116]: im seeing some of the same changes in both PRs. Typically we don't want to do that. The PRs could conflict with each other.

File: src/App.css (hunk 1 above)
Comment (RocketKevin) [reply src/App.css]: Sorry, they should be put in the same pull request

File: src/App.js (hunk 2 above)
Comment (RocketKevin) [reply src/App.js]: Sorry, they should be put in the same pull request

**Pull request merged**